thoth/storages/graph/postgres.py

```diff
@@ -5001,6 +5001,48 @@ def sync_dependency_monkey_result(self, document: dict) -> None:
 
                 inspection_run.dependency_monkey_run_id = dependency_monkey_run.id
 
+    def get_python_package_required_symbols(
+        self, package_name: str, package_version: str, index_url: str
+    ) -> List[str]:
+        """Get required symbols for a Python package in a specified version."""
+        package_name = self.normalize_python_package_name(package_name)
+        package_version = self.normalize_python_package_version(package_version)
+
+        query = self._session.query(PythonPackageVersion)
+
+        query = (
+            query.filter(PythonPackageVersion.package_name == package_name)
+            .filter(PythonPackageVersion.package_version == package_version)
+            .join((HasArtifact, PythonPackageVersion.python_artifacts))
```

Comment:
It's not `PythonPackageVersion` but `PythonPackageVersionEntity` that has `python_artifacts` because PackageAnalyzer and PackageExtract cannot sync in `PythonPackageVersion` but in `PythonPackageVersionEntity` as constraint we have in the syncs.